lib/code_corps/emails/forgot_password_email.ex

```diff
@@ -0,0 +1,20 @@
+defmodule CodeCorps.Emails.ForgotPasswordEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.Emails.BaseEmail
+
+  def create(user, token) do
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), [link: link(token)])
+  end
+
+  defp template_id, do: Application.get_env(:code_corps, :postmark_forgot_password_template)
+
+  defp link(token) do
+    Application.get_env(:code_corps, :site_url)
+    |> URI.merge("password/reset?token=#{token}")
```

Comment:
@begedin In order for the model hook on the ember side to consume this token, we could do something like `this.route('reset', { path: '/reset/:token_id'})` and get it in the params of the model hook.  What do you think about this change?  This could change this to `URI.merge("password/reset/#{token}")`